docs/src/main/asciidoc/user-guide/topics/import_files_to_panel.adoc

```diff
@@ -0,0 +1,284 @@
+            },
+            "contact": {
+                "firstName": "james",
+                "lastName": "smith",
+                "phone": "512-123-1234"
+            },
+            "orderId": "123"
+        },
+        "primitives": {
+            "stringPrimitive": "some value",
+            "booleanPrimitive": true,
+            "numberPrimitive": 24
+        },
+        "addressList": [
+            { "street": "123 any st", "city": "Austin", "state": "TX", "zip": "78626" },
+            { "street": "123 any st", "city": "Austin", "state": "TX", "zip": "78626" },
+            { "street": "123 any st", "city": "Austin", "state": "TX", "zip": "78626" },
+            { "street": "123 any st", "city": "Austin", "state": "TX", "zip": "78626" }
+        ]
+    }
+```
+* XML Schema Instance File: *XMLSchemaInst.xml*
+
+```
+ <ns:XmlOE xmlns:ns="http://atlasmap.io/xml/test/v2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://atlasmap.io/xml/test/v2 atlas-xml-test-model-v2.xsd ">
+    <ns:orderId>ns:orderId</ns:orderId>
+    <ns:Address>
+      <ns:addressLine1>ns:addressLine1</ns:addressLine1>
+      <ns:addressLine2>ns:addressLine2</ns:addressLine2>
+      <ns:city>ns:city</ns:city>
+      <ns:state>ns:state</ns:state>
+      <ns:zipCode>ns:zipCode</ns:zipCode>
+    </ns:Address>
+    <ns:Contact>
+      <ns:firstName>ns:firstName</ns:firstName>
+      <ns:lastName>ns:lastName</ns:lastName>
+      <ns:phoneNumber>ns:phoneNumber</ns:phoneNumber>
+      <ns:zipCode>ns:zipCode</ns:zipCode>
+    </ns:Contact>
+  </ns:XmlOE>
+```
+Using the same procedure as before, these files may be imported into either the *Source* or *Target*
+panel.
+
+There does exist one other method for establishing mappable fields within the AtlasMap data mapper.
+A Java class can be established where each field is represented as a class-wide public entity.  Arrays
+and data types are more discretely defined.  For example:
+
+* Java File: *Bicycle.java*
+```
+package io.paul;
+import io.paul.GeoLocation;
+
+public class Bicycle {
+    public int cadence;
+    public int gear;
+    public int speed;
+    public float[] seatHeight;
+    public String[] color;
+    public GeoLocation geoLocation;
+}
+```
+* Java File: *GeoLocation.java*
+```
+package io.paul;
+
+public class GeoLocation {
+    double lattitude;
+    double longitude;
+}
+```
+Compile the Java files and assemble the results into a Java archive file (i.e. Bicycle.jar).  Here's a hint:
+```
+javac -cp io.paul:. -d . GeoLocation.java Bicycle.java 
+jar cvf ../Bicycle.jar *
+```
+Now you can import the Bicycle.jar archive into AtlasMap.  The import procedure is slightly different for
+Java class archives.  You must:
+
+Import the file into AtlasMap.  Select the import icon *on the main tool bar*, not on
+the panel.
+
+image:ImportJavaClass1.png[Import a Java Class Archive File to Source Panel]
+
+An *Open File* dialog appears.  Select the location for Bicycle.jar.
```

Comment:
... for the `Bicycle.jar` file."
OR should this be:
Navigate to the `Bicycle.jar` file and select it." ???